### PR TITLE
Added VoiceOver localization for the compose placeholder

### PR DESCRIPTION
**Pull request by martijnvdwal**

### CODE REVIEW

**Comment by martijnvdwal** [comment]
fixes #170

File: Sources/Parley/Views/ParleyComposeView/ParleyComposeView.swift [hunk 1]
```diff
@@ -99,10 +99,16 @@ public class ParleyComposeView: UIView {
     var placeholder: String? {
         didSet {
             placeholderLabel.text = placeholder
-            textView.accessibilityLabel = placeholder
+            textView.accessibilityLabel = placeholderVoiceOver ?? placeholder
```
Comment (alexkok) [inline Sources/Parley/Views/ParleyComposeView/ParleyComposeView.swift:102]: Shouldn't we just do a `replace("...", "")`?

Introducing a new string doesn't feel needed for this one

**Review by alexkok — CHANGES_REQUESTED** [review]

File: Sources/Parley/Views/ParleyComposeView/ParleyComposeView.swift (hunk 1 above)
Comment (alexkok) [reply Sources/Parley/Views/ParleyComposeView/ParleyComposeView.swift:102]: Nevermind

**Comment by alexkok** [comment]
Released in [4.2.4](https://github.com/parley-messaging/ios-library/releases/tag/4.2.4) 🚀

**Pull request merged**